File: Iteration2/SourceCode/src/main/java/Semester.java
import java.util.ArrayList;

public class Semester {
    private ArrayList<Course> courseList = new ArrayList();
    int semesterID;
    public Semester(ArrayList courseList,int semesterID){//
        this.semesterID = semesterID;
        this.courseList = courseList;
    }

    public int getSemesterID() {
        return semesterID;
    }//

    public void setSemesterID(int semesterID) {
        this.semesterID = semesterID;
    }//

    public ArrayList<Course> getCourseList() {
        return courseList;
    }

    public void setCourseList(ArrayList<Course> courseList) {
        this.courseList = courseList;
    }
}

File: Iteration3/SourceCode/Semester.py
class Semester(object):
    def __init__(self, courseList, semesterID):


        self.semesterID = semesterID
        self.__courseList = courseList

    def getSemesterID(self):
        return self.semesterID

    def setSemesterID(self, semesterID):
        self.semesterID = semesterID

    def getCourseList(self):
        return self.__courseList

    def setCourseList(self, courseList):
        self.__courseList = courseList